ui/app/src/components/ActivityDashlet/ActivityDashlet.tsx

```diff
@@ -559,7 +559,10 @@ export function ActivityDashlet(props: ActivityDashletProps) {
 											color="primary"
 											size="small"
 											onClick={loadNextPage}
-											aria-label={formatMessage({ defaultMessage: 'Load {limit} more' }, { limit })}
+											aria-label={formatMessage(
+												{ id: 'activityDashlet.loadMore', defaultMessage: 'Load {limit} more' },
```

Comment:
Why add the ID?

Reply:
Those IDs are the ones you told me to add if there was the same message with an already existing id (e.g. see title of the Tooltip above with the FormattedMessage with an id). Same with the other occurrences.